analyzer/src/functTest/java/org/jboss/gm/analyzer/alignment/SimpleProjectFunctionalTest.java

```diff
@@ -211,4 +220,134 @@ public void verifySnapshotHandling() throws IOException, URISyntaxException, Man
+        assertThat(alignmentModel).isNotNull();
+        final Path jsonFile = projectRoot.resolve(Configuration.REPORT_JSON_OUTPUT_FILE);
+        assertThat(jsonFile).doesNotExist();
+        final Path textFile = projectRoot.resolve(System.getProperty("reportTxtOutputFile"));
+        assertThat(textFile).isRegularFile().isReadable();
+        assertThat(Files.readAllLines(textFile)).containsExactly(
+                "------------------- project org.acme.gradle:root:1.0.1",
+                "\tProject version : 1.0.1 --> 1.0.1.redhat-00002",
+                "",
+                "\tDependencies : org.acme.gradle:root:1.0.1 --> org.acme.gradle:root:1.0.1.redhat-00002",
+                "\tDependencies : com.yammer.metrics:metrics-core:2.2.0 --> com.yammer.metrics:metrics-core:2.2.0-redhat-00001",
+                "\tDependencies : org.hibernate:hibernate-core:5.3.7.Final --> org.hibernate:hibernate-core:5.3.7.Final-redhat-00001",
+                "\tDependencies : io.undertow:undertow-core:2.0.15.Final --> io.undertow:undertow-core:2.0.15.Final-redhat-00001",
+                "");
+    }
+
+    @Test
+    public void verifyAlignmentReportTextReportNonAligned() throws IOException, URISyntaxException {
+        System.setProperty("reportJSONOutputFile", "");
+        System.setProperty("reportTxtOutputFile", "alignmentReport.txt");
+        System.setProperty("reportNonAligned", Boolean.TRUE.toString());
+        final Path projectRoot = tempDir.newFolder("simple-project").toPath();
+        assertThat(projectRoot).isDirectory();
+        final TestManipulationModel alignmentModel = TestUtils.align(projectRoot.toFile(),
+                projectRoot.getFileName().toString());
+        assertThat(alignmentModel).isNotNull();
+        final Path jsonFile = projectRoot.resolve(Configuration.REPORT_JSON_OUTPUT_FILE);
+        assertThat(jsonFile).doesNotExist();
+        final Path textFile = projectRoot.resolve(System.getProperty("reportTxtOutputFile"));
+        assertThat(textFile).isRegularFile().isReadable();
+        assertThat(Files.readAllLines(textFile)).containsExactly(
+                "------------------- project org.acme.gradle:root:1.0.1",
+                "\tProject version : 1.0.1 --> 1.0.1.redhat-00002",
+                "",
+                "\tDependencies : org.acme.gradle:root:1.0.1 --> org.acme.gradle:root:1.0.1.redhat-00002",
+                "\tDependencies : com.yammer.metrics:metrics-core:2.2.0 --> com.yammer.metrics:metrics-core:2.2.0-redhat-00001",
+                "\tDependencies : org.hibernate:hibernate-core:5.3.7.Final --> org.hibernate:hibernate-core:5.3.7.Final-redhat-00001",
+                "\tDependencies : io.undertow:undertow-core:2.0.15.Final --> io.undertow:undertow-core:2.0.15.Final-redhat-00001",
+                "",
+                "\tNon-Aligned Dependencies : junit:junit:4.12",
+                "\tNon-Aligned Dependencies : org.apache.commons:commons-lang3:3.8.1",
```

Comment:
Can these be sorted alphabetically?

Reply:
Lets ignore this for now; it would be nice as an improvement but could be done in subsequent work (as PME doesn't sort either)